src/scss/skin-modern/components/_errormessageoverlay.scss

```diff
@@ -18,11 +18,14 @@
     right: 3em;
     text-align: center;
     user-select: text;
+    white-space: pre-line; // enable linebreak in text
 
-    // Vertically center the label (not exactly, but works for single line errors)
+    // Vertically center the label
     & {
-      margin-top: -1em;
+      // sass-lint:disable no-vendor-prefixes
+      -ms-transform: translateY(-50%); // required for IE9
```

Comment:
Not necessary. This is automatically added by the `autoprefixer` during the build.

Reply:
I checked this but `-ms` was not added for the `transform` property.

Reply:
@protyposis our config says `Last 2 versions` and IE9 is not included there so it will not be added in this case.

Reply:
Then we might have to update the config. 2 versions seem a bit low anyway given the fast release cycle of Chrome & FF. Can you check what the currently recommended config is?

Reply:
I think it's fair to drop IE9 support in UI v3 / Player v8 more than 2.5 years after Microsoft stopped supporting it: https://www.microsoft.com/en-us/windowsforbusiness/end-of-ie-support

Reply:
Even better 👍